A hostname should be fixed to allow easier build process.
	by Sergiusz Pawlowicz <debian@pawlowicz.name>
--- a/do
+++ b/do
@@ -6,7 +6,7 @@
 
 version=`cat version`
 project=nacl
-shorthostname=`hostname | sed 's/\..*//' | tr -cd '[a-z][A-Z][0-9]'`
+shorthostname="debianbuildhost"
 
 top="`pwd`/build/$shorthostname"
 bin="$top/bin"
